src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/PlanControllerTest.kt

```diff
@@ -26,19 +25,12 @@ import java.util.UUID
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class PlanControllerTest : IntegrationTestBase() {
 
-  @Autowired
-  lateinit var planRepository: PlanRepository
-  lateinit var planUuid: UUID
+  val staticPlanUuid = "556db5c8-a1eb-4064-986b-0740d6a83c33"
+  val mutablePlanUuid = "4fe411e3-820d-4198-8400-ab4268208641"
```

Comment:
I think there are a couple of reasons:

1. This sets the data to a guaranteed known state, with known values, bypassing the fact that business logic which may affect the data you think you're storing may change over time
2. It centralises creation of test data so that it's possible to get a clear view of what data should exist in the database, with relationships, at a point in time, without having to review all the executable test code. 

The latter can be mitigated by tearing down and adding data at the start of each test (although this is time-expensive), but I don't think the former can.